package.json

```diff
@@ -5,14 +5,21 @@
   "main": "lib/objection.js",
   "license": "MIT",
   "scripts": {
+    "precommit": "lint-staged",
```

Comment:
if you want to add a "lint" script, that'd be nice, but I think precommit is going to be a PITA.

I'd +1 the following:
```json
  "prettier": "prettier --write",
```